src/aliceVision/sfmData/View.hpp

```diff
@@ -303,6 +303,17 @@ class View
     return -1;
   }
 
+  /**
+   * @brief Get the corresponding "Orientation" metadata value
+   * @return the metadata value int or -1 if no corresponding value
+   */
+    int getMetadataOrientation() const
```

Comment:
I would suggest, for the sake of code readability, to use a enum for the rotation values.
According to this https://www.impulseadventure.com/photo/exif-orientation.html
the rotation metadata is encoded in 8 values, from 1 to 0. We could have a enum that better explains and map these values either in numerical degrees of the rotation needed to bring the image in the right position
(eg, enum class ImageRotation( 90 180 -90 etc) or even some words that are intuitive to get the right rotation.
This would much improve the code, instead of having 1-8 values for which one has to remember or check the correspondent rotation, we have a piece of more readable information.
What do you think @fabiencastan @FabienDanieau

Reply:
or more simply having (also) a function
```c++
AngleAxis<float> getMetadataOrientation() const
```

Reply:
Sounds good, I will do that.